.travis.yml

```diff
@@ -23,6 +23,7 @@ env:
     - DAQ_TEST=many
     - DAQ_TEST=aux
     - DAQ_TEST=topo
+    - DAQ_TEST=bacext
```

Comment:
Should not add a new entry here, it needs to be part of one of the existing test_ scripts.